### ISSUE TITLE
[Bug]: local folder are deleted by sync with read-only groupfolder

### ISSUE THREAD

**Issue by HFlor** [nextcloud/desktop#7518]
### ⚠️ Before submitting, please verify the following: ⚠️

- [x] This is a **bug**, not a question or a configuration issue.
- [x] This issue is **not** already reported on Github (I've searched it).
- [x] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

without any question the local folder are deleted

### Steps to reproduce

1. Creation of a group folder without write permission (via group)
2. A user in the group used above adds the folder synchronization with a local folder with subfolders and files
3. all local subfolders are deleted, only the files in the local folder (top level) are still there.

### Expected behavior

at least a warning that all data should be deleted should be displayed

### Which files are affected by this bug

no log entry

### Operating system

Windows

### Which version of the operating system you are running.

Windows 10

### Package

Official Windows MSI

### Nextcloud Server version

20.0.2

### Nextcloud Desktop Client version

3.14.3

### Is this bug present after an update or on a fresh install?

Fresh desktop client install

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [x] Default internal user-backend
- [x] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Nextcloud Server logs

```shell

```

**Comment by Rello** [comment]
Hello @HFlor 

I am not sure about this one
`A user in the group used above adds the folder synchronization with a local folder with subfolders and files`

do you mean that you select the group folder to be synced - but locally there was already a folder with that name and this one is then cleared?
If this is the case - then this is by design. Sync is sync. and if the server contains less files, the local version will be mirrored. 

let me know if there is a misunderstanding

**Comment by HFlor** [comment]
No, it is when setting up a new synchronization of the local data. This has never been set up as a sync target. I recorded it:
https://youtu.be/UE6onK7TMjc

**Comment by Rello** [comment]
Hello,

the client was enhanced significantly since the opening of this ticket.
You can help us with a retest.
If you are still encountering issues, please get back to us
